test/test_other.py

```diff
@@ -13389,3 +13389,44 @@ def test_node_pthreads_err_out(self):
   def test_windows_batch_file_dp0_expansion_bug(self):
     create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
     self.run_process(['build_with_quotes.bat'])
+
+  @parameterized({
+    'memory64_bigint': (True, True),
+    'bigint': (False, True),
+    'memory64': (True, False),
+    '': (False, False)
+  })
+  def test_add_js_function_64_bit(self, memory64, with_bigint):
+    if with_bigint:
+      self.set_setting('WASM_BIGINT')
+      self.node_args += shared.node_bigint_flags()
+
+    if memory64:
+      self.require_v8()
+      self.v8_args += ['--experimental-wasm-memory64']
+    else:
+      self.use_all_engines = True
+    self.set_setting('DEFAULT_LIBRARY_FUNCS_TO_INCLUDE', ['$addFunction'])
+    self.set_setting('RESERVED_FUNCTION_POINTERS')
+    self.set_setting('ENVIRONMENT', 'shell,node')
```

Comment:
I still see failures even with `require_wasm64`...

Reply:
`requires_v8` should take care of setting `ENVIRONMENT`... can you try again?

Reply:
Indeed it works. Don't know why I thought it didn't work before -- I suspect it had to do with the fact that I was also testing it with node since `require_v8` sets `ENVIRONMENT` to `shell`, but I had set it to `shell,node`.